clusterloader2/pkg/measurement/common/simple/profile.go

```diff
@@ -35,6 +35,74 @@ func init() {
 	measurement.Register(memoryProfileName, createMemoryProfileMeasurement)
 	measurement.Register(cpuProfileName, createCPUProfileMeasurement)
 }
+type profileConfig struct {
+	componentName string
+	provider      string
+	host          string
+	kind          string
+}
+
+func createProfileConfig(config *measurement.MeasurementConfig) (*profileConfig, error) {
+	pc := &profileConfig{}
+	if pc.componentName, err = util.GetString(config.Params, "componentName"); err != nil {
+		return nil, fmt.Errorf("invalid componentName field")
```

Comment:
NIT: Returning the original error might be better, it includes the name of the field you tried to access together with error details.
 Here and in the other places.

Reply:
done